server/controllers/users.js

```diff
@@ -23,6 +24,23 @@ usersRouter.post('/', async (request, response, next) => {
 
   try {
     const savedUser = await user.save()
+
+    // Create default categories on registration
+    const all = new Category({
+      name: 'All',
+      icon: 'all_icon',
+    })
+    const breakfast = new Category({
+      name: 'Breakfast',
+      icon: 'breakfast_icon',
+    })
```

Comment:
When the amount of default categories grows big, they could probably be moved somewhere as constants.